### PR TITLE
Solutions

### PR DESCRIPTION
**Pull request by yeli-buonya**
Update all assignment solutions. All tests are passing. I have not refactored assignments 1 & 2

### CODE REVIEW

**Comment by santiagobasulto** [comment]
This PR will be reviewed by @carasolomon

File: football_dictionaries/assignment_1.py [hunk 1]
```diff
@@ -1,2 +1,30 @@
...
+        formatted_list.append(player)
+
+    # iterate over the squads list array
+    for index, player in enumerate(squads_list):
+        # iterate through the players in squads_list, and labels
+        # updates the formatted list with individual player info
+        for info, label in zip(player, labels):
+            formatted_list[index].update({label: info})
+
+    return formatted_list
```
Comment (carasolomon) [inline football_dictionaries/assignment_1.py]: Very nice. I like how you implemented different functions such as enumerate() and zip() to accomplish this assignment. The labels technique is very clever as well.

File: football_dictionaries/assignment_2.py [hunk 2]
```diff
@@ -1,2 +1,18 @@
...
+    for player in formatted_list:
+        position = player.get('position')
+        # adds a field position
+        positions_list.setdefault(position, [])
+        # adds the player to their respective list
+        # by position
+        positions_list[position].append(player)
+
+    return positions_list
+
```
Comment (carasolomon) [inline football_dictionaries/assignment_2.py]: 👍

File: football_dictionaries/assignment_3.py [hunk 3]
```diff
@@ -1,2 +1,22 @@
...
+        # iterate through each country and check it against
+        # each country that the player is associated with
+        # if there's a match add that players position to the
+        # country, and passing an empty array as the default
+        for country in countries:
+            if (player['country'] == country):
+                countries[country].setdefault(player['position'], [])
+                countries[country][player['position']].append(player)
+
+    return countries
```
Comment (carasolomon) [inline football_dictionaries/assignment_3.py]: I like how your code is commented and explains the processes that are expected.

**Comment by santiagobasulto** [comment]
This PR will be reviewed by @amangrum2003

**Pull request closed without merging**